docs/libcurl/opts/CURLOPT_SEEKDATA.3

```diff
@@ -39,7 +39,7 @@ HTTP, FTP, SFTP
 .nf
 static int seek_cb(void *userp, curl_off_t offset, int origin)
 {
-  struct data *d = (struct data *)userp;
+  struct data *our_fd = (struct data *)userp;
   lseek(our_fd, offset, origin);
```

Comment:
This fixes one problem but this now instead passes in a `struct data` pointer to lseek() which also seems wrong.

How about we we instead change the lseek() call to become:
~~~c
lseek(d->our_fd, offset, origin);